validator/client/validator.go

```diff
@@ -1266,8 +1266,8 @@ func (v *validator) FindHealthyHost(ctx context.Context) bool {
 		if v.nodeClient.IsHealthy(ctx) { // healthy → done
 			return true
 		}
-		if len(v.beaconNodeHosts) == 1 {
-			log.WithField("host", v.Host()).Info("Beacon node is not responding, no backup node configured")
+		if len(v.beaconNodeHosts) == 1 && features.Get().EnableBeaconRESTApi {
```

Comment:
I was wrong in the assumption that gRPC doesn't have node hosts  i guess it's still populated because we can have both, so i will only display this log if it's for beacon rest

Reply:
But why display it only for REST? Doesn't the log make sense for grpc too?